src/codegen/updating-llvm.md

```diff
@@ -43,7 +46,9 @@ to the branch we're already using. The steps for this are:
 4. Check out the appropriate branch (typically named `rustc/a.b-yyyy-mm-dd`)
 5. Cherry-pick the upstream commit onto the branch
 6. Push this branch to your fork
-7. Send a Pull Request to rust-lang/llvm-project to the same branch as before
+7. Send a Pull Request to rust-lang/llvm-project to the same branch as before.
+   Be sure to reference the Rust and/or LLVM issue that you're fixing in the PR
+   description.
 8. Wait for the PR to be merged
 9. Send a PR to rust-lang/rust updating the `src/llvm-project` submodule with
    your bugfix
```

Comment:
This is another place to mention `git submodule update --remote src/llvm-project`, assuming the branch name in `.gitmodules` is current.